controlplane/kubeadm/internal/workload_cluster_etcd.go

```diff
@@ -49,6 +48,8 @@ func (w *Workload) EtcdIsHealthy(ctx context.Context) (HealthCheckResult, error)
 
 	expectedMembers := 0
 	response := make(map[string]error)
+	var memberList []*etcd.Member
+	inconsistentMemberList := false
```

Comment:
Might be using a positive name, so we should not negate down in the code
```suggestion
	nodesResportSameMemberList := true
```